client/core/core_test.go

```diff
@@ -6080,6 +6080,7 @@ func TestParseCert(t *testing.T) {
 
 func TestPreOrder(t *testing.T) {
 	rig := newTestRig()
+	defer rig.shutdown()
```

Comment:
This is pretty inconsequential as the last test, but added anyway.  Although I wonder if there's an issue with this test still given the logs:

```
[TRC] TCORE: notify: |DATA| (walletstate)
[DBG] TCORE[book]: Processing 0 cached order notes
[ERR] TCORE[book]: notification received out of sync. 1 != 0 - 1
[ERR] TCORE[book]: notification received out of sync. 1 != 0 - 1
[INF] TCORE: Connecting wallet for dcr
[ERR] TCORE: Error connecting to dcr wallet: test error
[INF] TCORE: Connecting wallet for btc
[ERR] TCORE: Error connecting to btc wallet: test error

```